mllib/src/test/scala/org/apache/spark/ml/stat/SummarizerSuite.scala

```diff
@@ -335,4 +335,65 @@ class SummarizerSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(Buffer.totalCount(summarizer) === 6)
   }
 
+  // TODO: this test should not be committed. It is here to isolate some performance hotspots.
+  test("perf test") {
+    val n = 10000000
+    val rdd1 = sc.parallelize(1 to n).map { idx =>
+      OldVectors.dense(idx.toDouble)
+    }
+    val trieouts = 10
```

Comment:
I think that 10 times without warmup is too small for performance measurement.
Can we use `Benchmark` class or add warmup run as `Benchmark` class does?